roles/confluent.kafka/tasks/main.yml

```diff
@@ -1,139 +1,139 @@
 - name: broker group
   group:
-    name: "{{kafka.broker.group}}"
+    name: "{{kafka.group}}"
```

Comment:
Recommend changing this to either 'broker' or 'kafka_broker'.

Reply:
kafka_broker